platform/darwin/src/MGLLocationManager.m

```diff
@@ -49,6 +57,22 @@ - (CLActivityType)activityType {
     return self.locationManager.activityType;
 }
 
+#if __IPHONE_OS_VERSION_MAX_ALLOWED >= 140000
+- (CLAccuracyAuthorization)accuracyAuthorization {
+    if (@available(iOS 14.0, *)) {
+        return self.locationManager.accuracyAuthorization;
+    } else {
+        return CLAccuracyAuthorizationFullAccuracy;
+    }
+}
+
+- (void)requestTemporaryFullAccuracyAuthorizationWithPurposeKey:(NSString *)purposeKey {
+    if (@available(iOS 14.0, *)) {
+        [self.locationManager requestTemporaryFullAccuracyAuthorizationWithPurposeKey:purposeKey];
```

Comment:
wouldn't this be an endless loop/recursive call if you are on iOS14. 
This looks like the implementation is calling the method itself over and over again, and if you are on iOS 14, that will happen non stop

Reply:
Same comment as above.